Src/AutoFakeItEasy/FakeItEasyRelay.cs

```diff
@@ -82,13 +82,28 @@ public object Create(object request, ISpecimenContext context)
 
             var fakeType = typeof(Fake<>).MakeGenericType(type);
 
-            var fake = context.Resolve(fakeType) as FakeItEasy.Configuration.IHideObjectMembers;
+            var fake = context.Resolve(fakeType);
             if (fake == null)

```

Comment:
You know the architecture better than I. The layered builders and relays boggle my mind, to tell the truth. If you're content to just know that `fake` has the quality that it `IsFake`, I'll remove the new stuff.